src/aap_eda/wsapi/consumers.py

```diff
@@ -148,19 +147,14 @@ def handle_heartbeat(self, message: HeartbeatMessage) -> None:
         ).first()
 
         if instance:
-            instance.status = ActivationStatus.RUNNING
             instance.updated_at = message.reported_at
-            instance.save(update_fields=["status", "updated_at"])
+            instance.save(update_fields=["updated_at"])
 
             activation = instance.activation
-            activation.status = ActivationStatus.RUNNING
-            activation.status_updated_at = message.reported_at
             activation.ruleset_stats[
                 message.stats["ruleSetName"]
             ] = message.stats
-            activation.save(
```

Comment:
if `status_updated_at` is removed from here, will `restart` be affected? it's used to calculate restart time interval, right?

Reply:
Restart logic depends on `status_updated_at` and `status == failed`. So if the status is still running the timestamp is not checked.